cogs/games.py

```diff
@@ -58,6 +60,144 @@ def is_lucky():
 
         await ctx.send(f'Question : {question}\nAnswer : {answer}')
 
+    @commands.command()
+    async def hangman(self, ctx):
+        def display_hangman(tries):
+            stages = [  # final state: head, torso, both arms, and both legs
+                """
+                   --------
+                   |      |
+                   |      O
+                   |     \\|/
+                   |      |
+                   |     / \\
+                   -
+                """,
+                # head, torso, both arms, and one leg
+                """
+                   --------
+                   |      |
+                   |      O
+                   |     \\|/
+                   |      |
+                   |     / 
+                   -
+                """,
+                # head, torso, and both arms
+                """
+                   --------
+                   |      |
+                   |      O
+                   |     \\|/
+                   |      |
+                   |      
+                   -
+                """,
+                # head, torso, and one arm
+                """
+                   --------
+                   |      |
+                   |      O
+                   |     \\|
+                   |      |
+                   |     
+                   -
+                """,
+                # head and torso
+                """
+                   --------
+                   |      |
+                   |      O
+                   |      |
+                   |      |
+                   |     
+                   -
+                """,
+                # head
+                """
+                   --------
+                   |      |
+                   |      O
+                   |    
+                   |      
+                   |     
+                   -
+                """,
+                # initial empty state
+                """
+                   --------
+                   |      |
+                   |      
+                   |    
+                   |      
+                   |     
+                   -
+                """
+            ]
+            return stages[tries]
+
+        def get_word():
+            word = random.choice(word_list)
+            return word.upper()
```

Comment:
There is no need for this to be a function, it's only referenced once in the code at line: 198, where you can simply use `word=random.choice(word_list).upper()`

Reply:
ok